requirements.txt

```diff
@@ -4,7 +4,8 @@ GitPython>=3.1.18
 joj-submitter>=0.0.8
 loguru>=0.5.3
 mattermostdriver>=7.3.2
+pandas>=2.0.0
 patool>=1.12
 pydantic>=2.0.2
 pydantic-settings>=2.0.1
-typer[all]>=0.3.2
+typer[all]>=0.3.2,<=0.11.1
```

Comment:
What is the breaking changes of later versions?

Reply:
typer >=0.12.0 does not support `all`

Reply:
Let's use the latest version here! `typer>=0.12.3`